src/commands/preboot-paired-simulator.yml

```diff
@@ -26,8 +39,11 @@ steps:
           ESCAPED_WATCHOS_VER=$(echo << parameters.watch-version >> | tr '.' '-')
           SIMLIST=$(xcrun simctl list -j)
           IPHONE_UDID=$(echo $SIMLIST | jq -r ".devices.\"com.apple.CoreSimulator.SimRuntime.iOS-$ESCAPED_IPHONE_VER\"[] | select(.name==\"<< parameters.iphone-device >>\").udid")
+          echo "export <<parameters.iphone-udid-var>>=$IPHONE_UDID" >> $BASH_ENV
```

Comment:
The parameter `iphone-udid-var` is referenced here but it is never declared. Maybe you meant: `iphone-device-udid-var` ?

Reply:
I did, thank you - fixed.